eventkit_cloud/jobs/models.py

```diff
@@ -134,7 +134,8 @@ class DataProvider(UIDMixin, TimeStampedModelMixin):
     """
     name = models.CharField(verbose_name="Service Name", unique=True, max_length=100)
     slug = LowerCaseCharField(max_length=40, unique=True, default='')
-    url = models.CharField(verbose_name="Service URL", max_length=1000, null=True, default='', blank=True)
+    url = models.CharField(verbose_name="Service URL", max_length=1000, null=True, default='', blank=True,
+                           help_text='The URL response time will affect the speed of data availability checks.')
```

Comment:
not the speed,.. how about?
> The SERVICE_URL is used as the endpoint for WMS, OSM, and WCS services.  It is also used as to check for availability for all OGC services.

Reply:
WFS, not WMS, right? Also there's a redundant "as".